src/config.ts

```diff
@@ -126,6 +126,15 @@ export interface Config {
   background?: string;
 
   /**
+   * The default visualization padding, in pixels, from the edge of the visualization canvas to the data rectangle. This can be a single number or an object with `"top"`, `"left"`, `"right"`, `"bottom"` properties.
+   *
+   * __Default value__: `5`
```

Comment:
I'm undecided about this too. 

For against `@default`:

1. For JSDoc this has different meaning.  

```
/** 
 *@default 
 */
const a = 5;   // This property will automatically 
```

That said, this is TSDoc. 

2. As we discussed that we should populate the markdown docs with text from TSDocs.  This will probably generate better output than `@default: 5`. 

But `@default` is good for JSON Schema.